docs/presentation/Common-Rules/MultiSchemaClassesSpecification.md

```diff
@@ -0,0 +1,33 @@
+# Multi Schema Classes Specification
```

Comment:
What are these `.snag` files under media folder?

Reply:
They're source files for screenshot .pngs. Keeping them around lets us avoid having to regenerate the whole screenshot if we just want to change callouts' style or reword something.

Reply:
What application would someone use to access the contents of these files?

Reply:
It's called "Snagit". You can get it from Software Center.

Reply:
While I could grab this software copy licensed by Bentley, outside collaborators do not have such privilege.

iTwin.js aims to be free ([see definition](https://www.gnu.org/philosophy/free-sw.en.html)), but hosting files on this repository that are restricted to proprietary software goes directly against that.

If all you need is to add some annotations and maybe later revise them, you could use any other freely available alternatives, or (my top pick) just add the annotations programmatically during documentation build.

Reply:
> just add the annotations programmatically during documentation build

I'd love that, but don't see how we could do this without spending weeks on it.

> you could use any other freely available alternatives

Right.. none of the alternatives I tried are actually usable for this kind of job. 

If delivering these files is a problem, then I'd say we don't deliver source of the screenshots at all and regenerate the whole screenshot if needed, or store them somewhere else internally. Personally I don't see this as a problem, because the files aren't used in building the documentation - the pngs are. @ColinKerr, @wgoehrig, do you have an opinion?

Reply:
> I'd love that, but don't see how we could do this without spending weeks on it.

I think you're hyperbolizing. I was able to accomplish rendering almost exact callouts in 2 hours, although there is a little more work to be done with automating text and arrow layout.

![result](https://user-images.githubusercontent.com/70327485/148986763-84be0db0-7eae-4561-9e8a-4facd101e427.png)

Reply:
Grigas had a chat with me. He told me no solutions are worth their time to not hot proprietary files on this repo.